### DISCUSSION TITLE
My Copilot Subscription expired earlier just 15 days not 1 month

### DISCUSSION THREAD

**Discussion by Mukeysh** [community/community#61027] [Copilot Conversations] [answered]
### Select Topic Area

Product Feedback

### Body

At the beginning of July, I made a payment of $10 for my subscription, which unfortunately expired after just 15 days. This had happened to me before when I had a subscription for only 8 days.

**Comment by jshtuazon2** [comment]
QDX-99677

**Comment by davecheney** [comment, marked as answer]
@Mukeysh please contact support, they can assist you https://support.github.com/contact.

**Reply by Mukeysh** [reply]
ok
